spec/integ/crypto/crypto.spec.ts

```diff
@@ -2385,6 +2450,96 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("crypto (%s)", (backend: string,
             expect(userSigningKey[secretStorageKey]).toBeDefined();
             expect(selfSigningKey[secretStorageKey]).toBeDefined();
         });
+
+        oldBackendOnly("should create a new megolm backup", async () => {
+            const backupVersion = "abc";
+            await bootstrapSecurity(backupVersion);
+
+            // Expect a backup to be available and used
+            const activeBackup = await aliceClient.getCrypto()!.getActiveSessionBackupVersion();
+            expect(activeBackup).toStrictEqual(backupVersion);
+        });
+
+        oldBackendOnly("Reset key backup should create a new backup and update 4S", async () => {
+            // First set up recovery
```

Comment:
"recovery" means nothing to me

```suggestion
            // First set up 4S and key backup
```

Reply:
Changed but we should stop considering 4S and keybackup as something different even if technically it is. The fact that we can have key backup without 4S is legacy. I think that account recovery is a better term

Reply:
> I think that account recovery is a better term

Well, it would be better if we said "account recovery" rather than just "recovery". Again, "recovery" on its own doesn't mean much to me.

But also: please let's not start using UX terms in code before we are *absolutely certain* they are going to stick. It is really confusing when code uses a term to mean something different from the UI, and that's not helped if it's a term we're still experimenting with.